go/vt/worker/split_clone.go

```diff
@@ -424,6 +435,29 @@ func (scw *SplitCloneWorker) Run(ctx context.Context) error {
 	return nil
 }
 
+func (scw *SplitCloneWorker) disableUniquenessCheckOnDestinationTablets(ctx context.Context) error {
```

Comment:
I believe this is still questionable. I don't see it used in this PR. Should we remove it until it becomes official?